freedomOfComputing/src/main/java/nl/tudelft/trustchain/FOC/util/ExtensionUtils.kt

```diff
@@ -0,0 +1,11 @@
+package nl.tudelft.trustchain.FOC.util
+
+class ExtensionUtils {
+    companion object {
+        const val apkExtenstion = "apk"
```

Comment:
You do not appear to use these consts throughout the project, please do as it makes the code more maintainable.